### ISSUE TITLE
PLE1206 is not raised when there is no arguments passed to the logging method call.

### ISSUE THREAD

**Issue by pavel-beaufort** [astral-sh/ruff#6942]
`PLE1206` (logging-too-few-args) error is not shown for such code snippet:
```
logger.error("%r: no filename.")
```
0.0.286

**Comment by charliermarsh** [comment]
Pylint doesn't raise on single-argument either. I'm not sure why.

**Comment by charliermarsh** [comment]
Okay, so I think this is intentional. If no message arguments are provided, the `logging` module treats the string as a literal. So this does not error:

```python
logging.error("%s %s")
```

(It logs the literal `%s %s`.)

However, this _does_ error:

```python
logging.error("%s %s", 1)
```

With:

```text
--- Logging error ---
Traceback (most recent call last):
  File "/Users/crmarsh/.local/share/rtx/installs/python/3.11.2/lib/python3.11/logging/__init__.py", line 1110, in emit
    msg = self.format(record)
          ^^^^^^^^^^^^^^^^^^^
  File "/Users/crmarsh/.local/share/rtx/installs/python/3.11.2/lib/python3.11/logging/__init__.py", line 953, in format
    return fmt.format(record)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/crmarsh/.local/share/rtx/installs/python/3.11.2/lib/python3.11/logging/__init__.py", line 687, in format
    record.message = record.getMessage()
                     ^^^^^^^^^^^^^^^^^^^
  File "/Users/crmarsh/.local/share/rtx/installs/python/3.11.2/lib/python3.11/logging/__init__.py", line 377, in getMessage
    msg = msg % self.args
          ~~~~^~~~~~~~~~~
TypeError: not enough arguments for format string
Call stack:
  File "/Users/crmarsh/workspace/ruff/foo.py", line 3, in <module>
    logging.error("%s %s", 1)
Message: '%s %s'
Arguments: (1,)
```

So, while it's a strange behavior, IMO it would be incorrect to raise an error here since omitting message arguments is a valid usage.

**Comment by dpinol** [comment]
I understand that logging does not crash with `logger.error("%s")`, but if ruff has rule PLE1206 equivalent to rule pylint E1206, I  guess it should have the same behaviour. So since pylint reports an error to `logger.error("%s")`, I guess that ruff should:
* Either have the same behaviour as pylint. Maybe disabling PLE1206 by default?
* Or clearly documenting the discrepancy wrt pylint
thanks
